src/firebase.ts

```diff
@@ -0,0 +1,15 @@
+import { initializeApp } from "firebase/app";
+import { connectAuthEmulator, getAuth } from "firebase/auth";
+
+const firebaseConfig = {
```

Comment:
according to ggogle answer here
https://stackoverflow.com/a/37484053
it's not a security risk to save it in public.
but it's very important to write the database rules well.